docs/integrations/sources/finage.md

```diff
@@ -0,0 +1,50 @@
+# Finage
+Real-Time Market Data Solutions for Stocks, Forex, and Crypto
+This connector can be used to extract data from various APIs such as symbol-list,Aggregates,Snapshot and Technical Indicators
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `type` | `string` | Type.  Indicator type. [ DEMA, EMA, SMA, WMA, RSI, TEMA, Williams, ADX ] |  |
+| `api_key` | `string` | API Key.  |  |
+| `market_type` | `string` | Market type. For Example us-stock, ca-stock, in-stock, ru-stock, forex, crypto, index |  |
+| `limit` | `string` | Limit. Limit of the news. Default is 10 and maximum allowed limit is 30. |  |
+| `from` | `string` | From. Start date |  |
+| `to` | `string` | To. End date |  |
+| `time` | `string` | Time. Time interval [ daily, 1min , 5min, 15min, 30min, 1hour, 4hour ] |  |
+| `period` | `string` | Period. Time period. Default is 10 |  |
+| `multiply` | `string` | Multiply. Time multiplier | 1 |
+| `time_aggregates` | `string` | Time aggregates. Size of the time. [minute, hour, day, week, month, quarter, year] |  |
```

Comment:
I tried to run the connector and it is not working for couple of endpoints.
There are a lot of parameters to configure this connector could be transformed to enum instead of being strings.
It looks trying to collect data from all stocks make impossible with free-tier or very expensive, maybe a better alternative is remove the symbol list and require the user to provide a list of symbols/stocks to retrieve data.